### PR TITLE
Add Fiji plugin Max_Inscribed_Circles

### PR DESCRIPTION
**Pull request by lldelisle**
Add Fiji plugin Max_Inscribed_Circles from BIOP
This is WIP because I wait for publication on maven.
----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by lldelisle** [comment]
@lacan
Would you mind to tell me what I should put as license and summary?
Thanks

**Comment by lacan** [comment]
Hi @lldelisle ,

The license is GPLv3.

Description could be something like this.

ImaheJ/ Fiji plugin implementing an iterative Largest Inscribed Circle algorithm which runs over a binary image or selection in order to fit the maximum number of non-touching circles down to a minimum diameter. 
The approach uses a Euclidean Distance Map in order to find candidate circles from the largest to the smallest, and is described here:
https://imagej.net/plugins/max-inscribed-circles

The code is written for Java 8, which is the currently supported Java version for Fiji and ImageJ.

Hope this works

Best

**Comment by lldelisle** [comment]
@lacan 
Could you tell me if it is correct for you?
The summary needs to be short while the description can be longer.

File: recipes/fiji-max-inscribed_circles/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  run:
+    - fiji>=20220414,!=h527b516_0,!=h9ee0642_0
+
+test:
+  files:
+    - test.groovy
+
+about:
+  home: https://imagej.net/plugins/max-inscribed-circles
+  license: GPLv3
```
Comment (bgruening) [inline recipes/fiji-max-inscribed_circles/meta.yaml:26]: can you please add a license_file to this recipe?

File: recipes/fiji-max-inscribed_circles/meta.yaml [hunk 2]
```diff
@@ -22,7 +22,15 @@ test:
     - test.groovy
 
 about:
-  home: https://imagej.net/Max_Inscribed_Circles
-  # TODO
-  license: GPL3
-  summary: This is an implementation of the Largest Inscribed Circle algorithm using an euclidean distance map.
```
Comment (lacan) [inline recipes/fiji-max-inscribed_circles/meta.yaml]: > This is an implementation of an iterative Largest Inscribed Circle algorithm using a euclidean distance map

**Comment by lacan** [comment]
Checked, with a suggestion. Thanks!

File: recipes/fiji-max-inscribed_circles/meta.yaml (hunk 1 above)
Comment (lldelisle) [reply recipes/fiji-max-inscribed_circles/meta.yaml:26]: done

File: recipes/fiji-max-inscribed_circles/meta.yaml (hunk 2 above)
Comment (lldelisle) [reply recipes/fiji-max-inscribed_circles/meta.yaml]: I put: `summary: ImageJ / Fiji plugin implementing an iterative Largest Inscribed Circle algorithm using a euclidean distance map`

Comment (lldelisle) [reply recipes/fiji-max-inscribed_circles/meta.yaml]: Is it ok?

**Comment by lldelisle** [comment]
@BiocondaBot please add label

**Review by bgruening — APPROVED** [review]

**Comment by lldelisle** [comment]
Is it ok to merge it?

**Comment by bgruening** [comment]
@BiocondaBot please merge

**Comment by bgruening** [comment]
you can use the bot to use it :)

**Comment by lldelisle** [comment]
Sorry, I thought I needed to be a member...

**Comment by lldelisle** [comment]
I feel stupid, I just realized that I mixed '-' and '_' in the name...
Should I leave it like this or change it now and create a duplicate but only this version?

**Comment by bgruening** [comment]
Mh, create a new PR with the new recipe, I can clean than up and remove package and container.

**Pull request merged**